common-module/src/main/java/domain/model/Category.java

```diff
@@ -0,0 +1,38 @@
+package domain.model;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import org.hibernate.annotations.CreationTimestamp;
+import domain.status.Exercise;
+
+@Entity
+public class Category {
+
+  @Id
+  @GeneratedValue
+  private long id;
+
+  @OneToOne(fetch = FetchType.LAZY)
+  private User user;
```

Comment:
하나의 category 가 여러 사용자(user) 들한테 매핑될 수 있으므로 ```OneToMany```가 맞을 것 같네요.
```suggestion
  @OneToMany(fetch = FetchType.LAZY, mappedBy = "category")
  private List<User> users = new ArrayList<>();
```